db/migrate/20250707070805_change_log_uid_to_string.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeLogUidToString < ActiveRecord::Migration[7.2]
+  def change
+    change_column :logs, :uid, :string
```

Comment:
Is this a straightforward change when there are rows with numbers?

We could preemptively truncate the table before applying the change, no?

Reply:
The integer uids will just be converted to the strings. So if a Log record exists with uid=1, after this migration it will just be uid="1". I'm relying on the log cleanup to eventually delete the outdated logs